### ISSUE TITLE
Can't connect phone & pc on two different subnets 

### ISSUE THREAD

**Issue by moonraker2** [GSConnect/gnome-shell-extension-gsconnect#1121]
Not sure if this is a bug so much as a limitation in the GSConnect/KDE connect software on pc and phone, respectively.
My phone and pc are on two different subnets/VLANs. This means they cannot connect.
However, if I put the phone on same subnet as the pc, they can successfully connect.

**Steps To Reproduce:**

1. Put phone and pc on different VLANs
2. Attempt connecting these two devices using GSConnect/KDE connect
3. The devices cannot find each other

**Desired behaviour**

For the devices to be able to connect even though they are on different VLANs.

**System Details (please complete the following information):**

GSConnect version 43. Installed from GNOME extensions website. 
Gnome version: 3.36.8
Distro: Ubuntu 20.04 LTS

The phone is a Samsung Galaxy S10 running latest available Android version from Samsung as well as the latest available version of KDE connect.

**Comment by moonraker2** [comment]
Upon further investigation it seems I am in fact able to connect pc and phone when manually providing the IP-address of the phone in GSConnect. That is, until I load the Private internet access VPN app, which seems to break it again. 

I have previously configured PIA app to "allow LAN traffic", but apparently I might also need to set up split-tunnel VPN feature to allow for the network communication between pc and phone to happen between VLANs.

Will update when I have confirmed whether this takes care of the connection issue.

**Comment by moonraker2** [comment]
Configuring split-tunnel vpn in PIA app fixed the issue.

**Comment by bb1950328** [comment]
I had a similar issue with my computer. It's connected to the internet via a wired network interface. My phone isn't connected to this network, but has a hotspot to which my computer is also connected via a USB WiFi stick. I ran `ifconfig` to find out the interface name of the WiFi stick (in my case `wlx200db045a6`). Then I ran `sudo arp-scan -I wlx200db045a6 --localnet` to find the IP address of my phone. Then I started "GSConnect Preferences", clicked on the burger menu and then on "Connect to...". Then I only had to enter the IP address of the phone into the entry and it worked!

(Just commenting here for other users that find this issue through a search engine and have the same problem)
